src/app/globals.css

```
@@ -79,3 +79,14 @@ body {
   color: var(--foreground);
   font-family: Arial, Helvetica, sans-serif;
 }
+
+@keyframes fadeSlideIn {
```

Stylelint (17.4.0)

[error] Expected keyframe name "fadeSlideIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)